packages/karma-build-scripts/createKarmaConfig.js

```diff
@@ -70,7 +70,7 @@ module.exports = function createKarmaConfig({ dirname, coverageExcludes, coverag
             [path.join(dirname, "test/index.ts")]: "webpack",
         },
         reporters: ["mocha", "coverage"],
-        singleRun: true,
+        singleRun: false,
```

Comment:
questionable edit... this is probably why the tests are hanging

Reply:
DERP.